editor/utils/heartbeat-post-locking.js

```diff
@@ -0,0 +1,83 @@
+/* global ajaxurl */
```

Comment:
Where does this global come from? and what's its value? In general we're trying to avoid globals entirely in the packages to keep them generic for npm consumption, So I'm trying to think of alternatives to avoid it.

Reply:
its localized by core, i think by virtue of enqueuing heartbeat. I can switch this to gutenberg's settings setup.